lib/compat/wordpress-6.1/class-gutenberg-rest-templates-controller.php

```diff
@@ -95,6 +95,75 @@ public function create_item( $request ) {
 		);
 	}
 
+	/**
+	 * Updates a single template.
+	 *
+	 * @since 5.8.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function update_item( $request ) {
+		$template = gutenberg_get_block_template( $request['id'], $this->post_type );
```

Comment:
I have only updated to use `gutenberg_get_block_template`. The function is copied from core.